client/src/modules/journal/templates/transaction_type.html

```diff
@@ -0,0 +1,3 @@
+<div class="ui-grid-cell-contents" translate>
```

Comment:
Do you need translate here?  Can you not use `hrText` instead and avoid translate?